plantuml/plantuml-server
# added docker build

# Description

Fixes the issue about docker images not generating on new releases.

## Type of change

- [x] Improvements (non-breaking runtime improvements)

## Testing Instructions

Recreating any of the existing tags should trigger a Github Action workflow that would generate new artefacts and push the necessary Docker images.

See [here](https://hub.docker.com/repository/docker/codeprefect/plantuml-server/tags?page=1&ordering=last_updated) for artefacts from the workflow on my fork.

@arnaudroques, it is important to configure the necessary docker configurations as secrets.

```
DOCKERHUB_USERNAME: plantuml-server
DOCKERHUB_TOKEN: obtain from DockerHub
```

## Checklist


[Fixes #184]
